### ISSUE TITLE
Rescue mode and Lua Scripts

### ISSUE THREAD

**Issue by David-VG** [dbasch/betaflight#1]
Hi, I am writing an small patch for MSP that sould allow to change the "GPS Rescue " settings with OpenTx with custom betaflight lua scripts.

The patch is done but will need to try it with the scripts, as soon as i get the custom scripts working with GPS Rescue will do a pull request.

Once all is ready you sould be able to change gpsRescue_t from your TX without a computer.

I think adding it to the OSD would also be great.

**Comment by s0up** [comment]
@David-VG Hey!  This is pretty awesome, we're still finalizing all of the settings though so probably worth adding this after we get it merged into master :P

**Comment by dbasch** [comment]
Agree, right now we're trying to get this into master. Shouldn't take long.
Once it's there, you should do the PR directly into BF master to make
things easier on them.
